src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Create;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Project\CreateProjectCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand\TableColumn;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Shared\Driver\Exception\Exception;
+use Keboola\StorageDriver\Teradata\ConnectionFactory;
+use Keboola\TableBackendUtils\Column\ColumnCollection;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\TableBackendUtils\Table\Teradata\TeradataTableQueryBuilder;
+
+final class CreateTableHandler implements DriverCommandHandlerInterface
+{
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof CreateTableCommand);
+
+        $db = ConnectionFactory::getConnection($credentials);
+
+        // root user is also root database
+        $databaseName = $credentials->getPrincipal();
+        $meta = $command->getMeta();
+        if ($meta !== null) {
+            // override root user and use other database as root
+            $meta = $meta->unpack();
+            assert($meta instanceof CreateProjectCommand\CreateProjectTeradataMeta);
+            $databaseName = $meta->getRootDatabase() === '' ? $databaseName : $meta->getRootDatabase();
+        }
+
+        // columns
+        $columns = [];
+        /** @var TableColumn $column */
+        foreach ($command->getColumns() as $column) {
+            /** @var CreateTableCommand\TableColumn\TeradataTableColumnMeta $columnMeta */
+            $columnMeta = $column->getMeta();
+            if ($columnMeta !== null) {
+                $columnMeta = $columnMeta->unpack();
+                assert($meta instanceof CreateTableCommand\TableColumn\TeradataTableColumnMeta);
+            }
+            $columnDefinition = new Teradata($column->getType(), [
+                'length' => $column->getLength(),
+                // TODO Syntax error, expected something like '(' between the 'varchar' keyword and ','.
+                'nullable' => $column->getNullable(),
+                // TODO Syntax error: expected something between the 'DEFAULT' keyword and ','.
+                'default' => $column->getUseDefault() ? null : $column->getDefault(),
```

Comment:
padaji mi SQL query, kdyz nastavim hodnoty `nullable` nebo `default`, viz hlasky v komentari.
Je ta Teradata v `table-backend-utils` a `php-datatypes` pripravena k pouziti nebo je potreba to jeste dodelat?

Reply:
uz to mam, pouzival jsem "nezname" typy jako `varchar` nebo `integer` a oni musi byt striktne `VARCHAR` a `INTEGER`.